src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/CaCaseloadServiceTest.kt

```diff
@@ -167,6 +174,12 @@ class CaCaseloadServiceTest {
     inner class `in the hard stop period` {
       @Test
       fun `Sets NOT_STARTED licences to TIMED_OUT when in the hard stop period`() {
+        whenever(caseloadService.prisonerToCaseloadItem(any(), anyOrNull())).thenReturn(
+          TestData.caseLoadItem().copy(
```

Comment:
prisonerToCaseloadItem(any(), anyOrNull())) this is the bit I change not sure if it should be null best check